test/Microsoft.AspNet.Razor.Test/Parser/Html/HtmlDocumentTest.cs

```diff
@@ -70,7 +70,11 @@ public void ParseDocumentCorrectlyHandlesOddlySpacedHTMLElements()
             ParseDocumentTest("<div ><p class = 'bar'> Foo </p></div >",
                 new MarkupBlock(
                     BlockFactory.MarkupTagBlock("<div >"),
-                    BlockFactory.MarkupTagBlock("<p class = 'bar'>"),
+                    new MarkupTagBlock(
+                        Factory.Markup("<p"),
+                        new MarkupBlock(
+                            Factory.Markup(" class")),
+                        Factory.Markup(" = 'bar'>")),
```

Comment:
why did change to support minimized attributes have any affect on how `MarkupTagBlock` is generated for non-minimized case?